code/_core/obj/item/weapon/ranged/laser/freezegun.dm

```diff
@@ -30,6 +31,39 @@
 
 	value = 1500
 
+
+	attachment_whitelist = list(
```

Comment:
Undocumented changes aaaaaaaaaaaaaaaaaaaaaaaaaaaa please remove these and put them in another PR

Reply:
wait these were already added though. Fuck sorry I'm still new to github.